com.unity.uiwidgets/Runtime/engine/UIWidgetsMessageManager.cs

```diff
@@ -74,18 +74,28 @@ public class UIWidgetsMessageManager: MonoBehaviour {
             }  
         }
 
+        Queue<string> messages = new Queue<string>();
```

Comment:
this is only need on Android ? if so we should use #define to wrap it in.

please also try to rename all Android specific variables/functions to a better name (to tell people this is android-only stuff)

Reply:
So far this is android only, but ios might need similar things as well. This is a temp solution. After this commit, I will work on channel and platform message.